app/src/noapi/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -24,7 +25,7 @@
 import javax.microedition.khronos.egl.EGLConfig;
 import javax.microedition.khronos.opengles.GL10;
 
-public class PlatformActivity extends Activity {
+public class PlatformActivity extends ComponentActivity {
```

Comment:
The HVR PlatformActivity also extends from Activity, we should migrate it as well.

Reply:
Done! Thanks!